shell/src/main/java/org/apache/kafka/shell/MetadataNodeManager.java

```diff
@@ -333,6 +339,34 @@ private void handleCommitImpl(MetadataRecordType type, ApiMessage message)
                 producerIds.create("nextBlockStartId").setContents(record.nextProducerId() + "");
                 break;
             }
+            case ACCESS_CONTROL_ENTRY_RECORD: {
+                AccessControlEntryRecord record = (AccessControlEntryRecord) message;
+                DirectoryNode acls = data.root.mkdirs("acl").mkdirs("id");
+                FileNode file = acls.create(record.id().toString());
```

Comment:
Just as KIP-778 described, each ACL is shown in /acl/id/<uuid> in its JSON form. however, I should mention here that their schema in ZkNode is more hierarchical:
```
get /kafka/kafka-acl/Topic/foo
 
{
    "version":1,
    "acls":[
        {
            "principal":"User:kafka",
            "permissionType":"Allow",
            "operation":"Read",
            "host":"1.1.1.1"
        }
    ]
}
```